tests/domain/allocine_test.py

```diff
@@ -50,11 +53,11 @@ def test_should_extract_movies_from_api_result(self):
                     "movie": {
                         "id": "TW92aWU6NTA0MDk=",
                         "internalId": 50609,
-                        "title": "Le Ch\u00e2teau ambulant"
+                        "title": "Le Ch\u00e2teau ambulant",
+                        "type": "SPECIAL_TYPE"
```

Comment:
D'après, le ticket `SPECIAL_TYPE` n'existe pas dans les cas possible.
Si ça te va, je remplacerais bien par un de la liste, ça évitera qu'on se demande ce qu'est un `special type`
```suggestion
                        "type": "BRAND_CONTENT"
```

Reply:
Une coquille ! Tu as raison